pkg/generators/cluster/managedclusterlabels.go

```diff
@@ -37,11 +37,24 @@ func GetManagedClusterLabelMetricFamilies(hubClusterID string) metric.FamilyGene
 				mangedClusterID,
 			}
 
-			regex := regexp.MustCompile(`[^\w]+`)
+			// Regex to match non-word characters or the first digit in the key
```

Comment:
could we add more comments here? with some example on how the key will be changed. Otherwise it will be really hard to track in the future.

Reply:
yes, add more comments and examples.